src/helpers/crypto.cpp

```diff
@@ -30,14 +31,19 @@ void cryptoSign(Sha256Hash hash, Ark::Crypto::Identities::PrivateKey privateKey,
   encodeDER(toDER(r_der), toDER(s_der), signature);
 }
 
-bool cryptoVerify(Ark::Crypto::Identities::PublicKey publicKey, Sha256Hash hash, std::vector<uint8_t>& signature)
-{
+bool cryptoVerify(Ark::Crypto::Identities::PublicKey publicKey, Sha256Hash hash,
+                  std::vector<uint8_t>& signature) {
   /* Get the Uncompressed PublicKey */
-  auto publicKeyBytes = publicKey.toBytes(); // compressed publicKey bytes (uint8_t*)
-  uint8_t uncompressedPublicKey[64] = {}; // create uncompressed publicKey buffer (uint8_t[64])
-  const struct uECC_Curve_t * curve = uECC_secp256k1(); // define the curve-type
-  uECC_decompress(publicKeyBytes, uncompressedPublicKey, curve); // decompress the key
-  if (uECC_valid_public_key(uncompressedPublicKey, curve) == 0) { return false; }; // validate the uncompressed publicKey
+  auto publicKeyBytes =
+      publicKey.toBytes();  // compressed publicKey bytes (uint8_t*)
+  uint8_t uncompressedPublicKey[64] =
+      {};  // create uncompressed publicKey buffer (uint8_t[64])
```

Comment:
Ones like these don't seem correct either. There's no reason for the comment to cause the line to be split up like this.

Reply:
I'm open to ideas.  We can make the max line length longer or we can shorten the comment.  We don't need to put the type in the comment.